src/lib/api/org/index.ts

```diff
@@ -179,9 +196,72 @@ export async function listProjects(
           JSON.stringify({ data: res.data, status: statusCode }),
       );
     }
-    return res.data || {};
+    
+    let v3responseData = res.data as v3ProjectsResponse
+    let projectData = convertToSnykProject(orgId.trim(), v3responseData)
+
+    while (v3responseData.links.next) {
```

Comment:
everything here is a repeat of above so any changes are needed we will do them twice and again for every API call we make, can it be 1  generic function that knows to handle rest API pagination? see example: 

https://github.com/snyk-tech-services/snyk-api-import/blob/master/src/lib/source-handlers/bitbucket-cloud/list-workspaces.ts#L19

1 function is called  `fetchAllProjects` and a second function that can fetch just 1 page `fetchProjects`, while there is a next page keep calling `fecthProjects` and it returns { projects, nextPageLink}` so you know when to stop + can keep appending project data to a running total array







```

while (hasNext) {

.... do the stuff

}



```